release-notes-8.1.0.md

```diff
@@ -1,8 +1,8 @@
-!not-ready-for-release!
-
 #### Version Number
 ${version-number}
 
 #### New Features
+-US990352: Include correlation id in custom data of outgoing worker task message.
```

Comment:
What's been done is pretty much the opposite of that.  It used to pass the correlation id along in the custom data.  With these changes it no longer does that.  It now relies on the Worker Framework's underlying functionality to pass the correlation id along.